dynamic_programming/sum_of_subset.py

```diff
@@ -22,14 +27,9 @@ def isSumSubset(arr, arrLen, requiredSum):
     # uncomment to print the subset
     # for i in range(arrLen+1):
     #     print(subset[i])
+    print(subset[arrLen][requiredSum])
```

Comment:
Unnecessary . Try this.
take test.py as
```
def returns_true():
    """
    >>> returns_true()
    True
    """
    return True
```

then
`python3 -m doctest test.py -v`